calibration/experiments/sphere_held_suarez_rhoe_equilmoist/model_config.yml

```diff
@@ -0,0 +1,15 @@
+dt: 500secs
+t_end: 560days
+ode_algo: ARS343
+moist: equil 
+forcing: held_suarez
+precip_model: 0M
+job_id: sphere_held_suarez_rhoe_equilmoist
+output_dir: output/sphere_held_suarez_rhoe_equilmoist
+restart_file: /groups/esm/ClimaArtifacts/artifacts/atmos_held_suarez_obs/day200.0.hdf5
```

Comment:
This will be a problem everywhere but the caltech cluster

Reply:
Maybe add instructions on how to fill this field properly if one wants to reproduce this